bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/cannon/Cannon.java

```diff
@@ -1,5 +1,125 @@
+/*
+ * File: cannon.java
+ * Class: Cannon
+ *
+ * Version: 0.0.3
+ * Date: September 12th, 2016
+ */
+
 package nl.tudelft.bust_a_move20162017.bust_a_move_framework.cannon;
 
+import nl.tudelft.bust_a_move20162017.bust_a_move_framework.bubble.Bubble;
+import nl.tudelft.bust_a_move20162017.bust_a_move_framework.bubble.BubbleGenerator;
+import nl.tudelft.bust_a_move20162017.bust_a_move_framework.game.Game;
+import org.newdawn.slick.Color;
+import org.newdawn.slick.Graphics;
+
+/**
+ * The Cannon class represents a cannon entity.
+ *
+ * @author Maurice Willemsen
+ */
+
 public class Cannon {
 
+	private int X;
+	private int Y;
+	private int SIZE;
+	private Color cannonColour;
+
+	private int ANGLE;
+	private Bubble currBubble;
+	private Bubble nextBubble;
+	private Game game;
+
+	public int TIME_SHOT_FIRED;
+
+	/**
+	 * Creates Cannon instance
+	 */
+
+	public Cannon(Game game) {
+		this.X = 320;
+		this.Y = 450;
+		this.SIZE = 40;
+		this.ANGLE = 0;
+		this.cannonColour = new Color(Color.red);
```

Comment:
I believe this can just be `this.cannonColour = Color.red` ... though I'm not 100%